### PR TITLE
Use ocamlformat

### PR DESCRIPTION
**Pull request by samoht**
There's still some weird things happening with the comments (especially the copyright headers) so I am not sure if that's a good idea to merge this yet.

### CODE REVIEW

File: src/irmin-chunk/irmin_chunk.mli [hunk 1]
```diff
@@ -1,99 +1,87 @@
...
- * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
- * WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
- * MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
- * ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
- * WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
- * ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
- * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
- *)
+(* * Copyright (c) 2013-2015 Thomas Gazagnaire <thomas@gazagnaire.org> *
+   Copyright (c) 2015 Mounir Nasr Allah <mounir@nasrallah.co> * * Permission to
```
[the remark this answers is not in the document]
Comment (gpetiot) [reply src/irmin-chunk/irmin_chunk.mli]: Yes it will

**Comment by Julow** [comment]
What do you think of adding `type-decl=sparse` ? Most type declarations are sparse in the current code.

**Comment by samoht** [comment]
No strong opinion, both works for me.

**Comment by samoht** [comment]
I've pushed an update with ocamlformat.0.9 and `--profile=conventional`

**Comment by samoht** [comment]
The result looks very good to me; WDYT @zshipko should we include this in 2.0?

**Comment by zshipko** [comment]
Yeah, I don't see any issue with including this from a brief scan of the changed files - I've been using `ocamlformat` on some other code and it's been working well for me!

**Comment by samoht** [comment]
ok let's try this then :-)

**Pull request merged**